pyproject.toml

```diff
@@ -51,7 +51,8 @@ line_length = 100
 
 [tool.pytest.ini_options]
 markers = [
-  "script: Marks tests provided as shell scripts",
+  "script: tests provided as shell scripts",
+  "help_page: tests that render help pages",
```

Comment:
Does this belong in this PR? Or should it be its own thing?

Reply:
It's not enough for me to bother making a separate PR for it...